src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -254,7 +254,7 @@ type EventContexts = {
   trace?: TraceContext;
 };
 
-type SentryEventBase = {
+export type SentryEventBase = {
```

Comment:
Why is this exported? Seems like this should be considered "private" and the `Event` type is the public version